osmaxx/core/templates/pages/about_us.html

```diff
@@ -18,7 +18,7 @@
     <h3>Useful links</h3>
     <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
 
-    <h3>About Us</h3>
+    <h3>About us</h3>
```

Comment:
Despite its name, sentence case can be used for titles (whether those titles are full sentences or not).

To quote the page you linked:
> This form, recommended by the Publication Manual of the American Psychological Association for titles in reference lists, is popular with many online and print publications. In fact it's now the standard form for titles and headlines in most countries